tests/unit/commands/test_remote.py

```diff
@@ -0,0 +1,103 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2019 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License remote_build 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from unittest import mock
+
+from testtools.matchers import Contains, Equals
+
+import snapcraft.internal.remote_build.errors as errors
+from . import CommandBaseTestCase
+from tests import fixture_setup
+
+
+class RemoteBuildTests(CommandBaseTestCase):
+    def setUp(self):
+        super().setUp()
+        self.useFixture(fixture_setup.SnapcraftYaml(self.path))
+
+    @mock.patch("snapcraft.cli.remote.LaunchpadClient")
+    def test_remote_build(self, mock_lc):
+        lc = mock_lc.return_value
```

Comment:
why don't you convert this into a fixtues.MockPatch in setup with all the default values, then just apply the variations in the tests which will make obvious what the change for the individual test is.

This particular assignment is also very confusing for a casual read,

Reply:
Will do

Reply:
Done!